bootstrap/fabric/src/main/java/org/geysermc/geyser/platform/fabric/GeyserFabricDumpInfo.java

```diff
@@ -43,21 +43,27 @@
 @Getter
 public class GeyserFabricDumpInfo extends BootstrapDumpInfo {
 
-    private String platformVersion = null;
+    private final String platformName;
+    private String platformVersion;
+    private final String minecraftVersion;
     private final EnvType environmentType;
 
     @AsteriskSerializer.Asterisk(isIp = true)
     private final String serverIP;
     private final int serverPort;
+    private final boolean onlineMode;
     private final List<ModInfo> mods;
 
     public GeyserFabricDumpInfo(MinecraftServer server) {
         FabricLoader.getInstance().getModContainer("fabricloader").ifPresent(mod ->
             this.platformVersion = mod.getMetadata().getVersion().getFriendlyString());
 
+        this.platformName = server.getServerModName();
+        this.minecraftVersion = server.getServerVersion();
```

Comment:
`platformName` at the top-top of the constructor :p sorry

Reply:
oh lol, sorry
done